test/e2e/upgrades/apps/daemonsets.go

```diff
@@ -126,11 +126,8 @@ func checkRunningOnAllNodes(f *framework.Framework, namespace string, selector m
 
 	nodeNames := make([]string, 0)
 	for _, node := range nodeList.Items {
-		if len(node.Spec.Taints) == 0 {
-			nodeNames = append(nodeNames, node.Name)
-		} else {
-			framework.Logf("Node %v not expected to have DaemonSet pod, has taints %v", node.Name, node.Spec.Taints)
-		}
+		// DaemonSet Pods are expected to run on all the nodes in e2e.
+		nodeNames = append(nodeNames, node.Name)
```

Comment:
Would you also print node taints when the test fails?

Reply:
done, give log when we ignore taints on nodes.